tools/dds/dds-server/rs-dds-server.cpp

```diff
@@ -94,205 +95,171 @@ void start_streaming( std::shared_ptr< tools::lrs_device_controller > lrs_device
     } );
 }
 
-void add_init_device_header_msg( rs2::device dev, std::shared_ptr<realdds::dds_device_server> &server )
-{
-    using namespace realdds::topics;
-   
-    device::notification::device_header_msg device_header_msg;
+void add_init_device_header_msg( rs2::device dev, std::shared_ptr< dds_device_server > &server )
+{   
+    topics::device::notification::device_header_msg device_header_msg;
     auto &&sensors = dev.query_sensors();
-    device_header_msg.num_of_sensors = sensors.size();
+    size_t num_of_streams = 0;
+    for (auto sensor : sensors)
+    {
+        num_of_streams += sensor.get_stream_profiles().size();
```

Comment:
That's the number of profiles, not the number of streams... right?

Reply:
Fixed